app/src/utils/validator/errorText.ts

```diff
@@ -0,0 +1,14 @@
+export default function getText(fieldName: string, arg?: string): string {
+    switch (fieldName) {
+        case 'required':
```

Comment:
Соответственно тут можно использовать `FIELD_NAMES`

```suggestion
        case FIELD_NAMES.required:
```